eng/pipelines/coreclr/templates/xplat-pipeline-job.yml

```diff
@@ -109,7 +109,7 @@ jobs:
     - librariesDownloadDir: ''
 
     - ${{ if ne(parameters.liveLibrariesBuildConfig, '') }}:
-      - librariesArtifactName: ${{ format('libraries_bin_{0}_{1}{2}_{3}_{4}', 'netcoreapp', parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.liveLibrariesBuildConfig) }}
+      - librariesArtifactName: ${{ format('libraries_bin_netcoreapp_{0}_{1}_{2}', parameters.osGroup, parameters.archType, parameters.liveLibrariesBuildConfig) }}
```

Comment:
Why are we removing the osSubgroup? wouldn't this break the linux_musl libraries builds?

Reply:
Oh sorry. I see this is just in coreclr's consumption, but this might break if you guys start depending on an artifact that contains ossubgroup like musl.

Reply:
Hmm, I was under the impression you yourself told me the other day that netcoreapp artifacts don't include OS subgroup information.